addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -161,18 +161,38 @@
                             <t t-set="line_counter" t-value="line_counter + 1"/>
                             <t t-call="l10n_it_edi.account_invoice_line_it_FatturaPA"/>
                         </t>
-                        <t t-foreach="record.line_ids.filtered(lambda line: line.tax_line_id)" t-as="tax_line">
-                            <DatiRiepilogo>
-                                <!--2.2.2-->
-                                <AliquotaIVA t-esc="format_numbers(tax_line.tax_line_id.amount)"/>
-                                <Natura t-if="tax_line.tax_line_id.l10n_it_has_exoneration" t-esc="tax_line.tax_line_id.l10n_it_kind_exoneration"/>
-                                <ImponibileImporto t-esc="format_monetary(tax_line.tax_base_amount, currency)"/>
-                                <Imposta t-esc="format_monetary(tax_line.price_unit, currency)"/>
-                                <EsigibilitaIVA t-if="not tax_line.tax_line_id.l10n_it_has_exoneration or tax_line.tax_line_id.l10n_it_kind_exoneration=='N6'" t-esc="tax_line.tax_line_id.l10n_it_vat_due_date"/>
-                                <RiferimentoNormativo t-if="tax_line.tax_line_id.l10n_it_has_exoneration" t-esc="tax_line.tax_line_id.l10n_it_law_reference"/>
-                            </DatiRiepilogo>
+                        <t t-if="record.line_ids.filtered(lambda line: line.tax_line_id)">
+                            <t t-foreach="record.line_ids.filtered(lambda line: line.tax_line_id)" t-as="tax_line">
```

Comment:
Something must have gone wrong here from v12 to v13 as the tax lines of 0 were not generated anymore.  However, there has to be a way here of filtering out the one tax that has exoneration instead of the all or nothing approach.  It would be better if we could already take care of the case where we have multiple taxes on the line (e.g. with withholdings, not supported yet)